<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fetch_episodeClass</class>
 <widget class="QMainWindow" name="fetch_episodeClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>578</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>媒体刮削器</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <widget class="QTableWidget" name="SeasonTable">
       <property name="editTriggers">
        <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
       </property>
       <property name="verticalScrollMode">
        <enum>QAbstractItemView::ScrollMode::ScrollPerPixel</enum>
       </property>
       <property name="horizontalScrollMode">
        <enum>QAbstractItemView::ScrollMode::ScrollPerPixel</enum>
       </property>
       <column>
        <property name="text">
         <string>文件夹</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>状态</string>
        </property>
       </column>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="PathLabel">
            <property name="text">
             <string>路径：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="PathEdit">
            <property name="text">
             <string/>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="BrowseButton">
            <property name="text">
             <string>打开</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLabel" name="TitleLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>标题：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="TitleText">
            <property name="text">
             <string/>
            </property>
            <property name="textInteractionFlags">
             <set>Qt::TextInteractionFlag::LinksAccessibleByMouse|Qt::TextInteractionFlag::TextSelectableByKeyboard|Qt::TextInteractionFlag::TextSelectableByMouse</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="SeasonLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>季度：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="SeasonText">
            <property name="text">
             <string/>
            </property>
            <property name="textInteractionFlags">
             <set>Qt::TextInteractionFlag::LinksAccessibleByMouse|Qt::TextInteractionFlag::TextSelectableByKeyboard|Qt::TextInteractionFlag::TextSelectableByMouse</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QLabel" name="LocalText">
              <property name="text">
               <string>本地文件</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QListWidget" name="LocalList">
              <property name="selectionMode">
               <enum>QAbstractItemView::SelectionMode::MultiSelection</enum>
              </property>
              <property name="verticalScrollMode">
               <enum>QAbstractItemView::ScrollMode::ScrollPerPixel</enum>
              </property>
              <property name="horizontalScrollMode">
               <enum>QAbstractItemView::ScrollMode::ScrollPerPixel</enum>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QPushButton" name="LocalUp">
                <property name="text">
                 <string>上移</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="LocalDown">
                <property name="text">
                 <string>下移</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="LocalAdd">
                <property name="text">
                 <string>添加</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="LocalRemove">
                <property name="text">
                 <string>删除</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QLabel" name="RemoteText">
              <property name="text">
               <string>对应数据</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QListWidget" name="RemoteList">
              <property name="selectionMode">
               <enum>QAbstractItemView::SelectionMode::MultiSelection</enum>
              </property>
              <property name="verticalScrollMode">
               <enum>QAbstractItemView::ScrollMode::ScrollPerPixel</enum>
              </property>
              <property name="horizontalScrollMode">
               <enum>QAbstractItemView::ScrollMode::ScrollPerPixel</enum>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QPushButton" name="RemoteUp">
                <property name="text">
                 <string>上移</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="RemoteDown">
                <property name="text">
                 <string>下移</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="RemoteAdd">
                <property name="text">
                 <string>添加</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="RemoteRemove">
                <property name="text">
                 <string>删除</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="SaveButton">
            <property name="text">
             <string>保存更改</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="WriteButton">
            <property name="text">
             <string>写入选中项</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="CoverCheck">
            <property name="text">
             <string>覆盖前确认</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="NextButton">
            <property name="text">
             <string>下一步</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
